http-clients/src/main/java/org/triplea/http/client/web/socket/messages/WebsocketMessageType.java

```diff
@@ -1,30 +1,29 @@
 package org.triplea.http.client.web.socket.messages;
 
 import com.google.common.base.Preconditions;
-import java.util.function.Consumer;
-import java.util.function.Function;
 
 /**
- * Interface representing a websocket message type that knows how to extract a payload object from a
- * {@code ServerMessageEnvelope} and also knows how to send that payload to a parameter listener.
+ * Interface representing a websocket message type that knows about the correct type to extract from
+ * a payload object from a {@code ServerMessageEnvelope}.
+ *
+ * <p>Delegates {@link #sendPayloadToListener(ServerMessageEnvelope, Object)} to {@link
```

Comment:
nit, implementation detail in this comment. More important to javadoc the purpose and intent of an interface. I think the commentary here is describing  a bit too much about how the class does what it does as opposed to what the class is supposed to do. Would probably also be good to have a javadoc describe expected contexts and usages where we would use the interface, what contracts we would be guaranteeing when implementing it.